src/BlobField/index.js

```diff
@@ -86,7 +86,7 @@ export default function BlobField({ collapsed = false }) {
       wrapper.removeEventListener("touchmove", handleTouchMove);
       window.removeEventListener("resize", handleResize);
     };
-  }, [blobsRef]);
+  }, [blobsRef, collapsed, history]);
```

Comment:
Pretty sure we don't actually want these here, even though the linter yells about it.
These values determine whether the function is called again when the vars change. We really only want this func to run once when the component is mounting... if it runs again it will reinitialize `Blobs` causing all the blobs to change position. I haven't tested, but my guess is adding `collapsed` here messes up the collapse animation.
Having `history` here is harmless, but also unnecessary because it never changes.

Reply:
You can use
```
//eslint-disable-next-line react-hooks/exhaustive-deps
```
to shut up the linter here